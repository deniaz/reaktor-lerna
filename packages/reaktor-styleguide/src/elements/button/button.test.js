import React from 'react';
import { create } from 'react-test-renderer';

import { Button } from '.';

describe('Element', () => {
  describe('Button', () => {
    it('matches snapshot', () => {
      const fn = jest.fn();
      const component = create(<Button label="Button" onClick={fn} />);
      const tree = component.toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
