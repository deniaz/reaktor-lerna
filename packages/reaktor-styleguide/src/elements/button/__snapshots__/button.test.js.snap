// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Element Button matches snapshot 1`] = `
<button
  className="button"
  disabled={false}
  onClick={[MockFunction]}
>
  Button
</button>
`;
